src/main/webapp/WEB-INF/tags/shared/feedbackSubmissionEdit/feedbackSubmissionEdit.tag

```diff
@@ -23,7 +23,22 @@
                     <h3 class="text-center">Previewing Session as ${isInstructor ? "Instructor" : "Student"} ${moderatedPersonName} (${moderatedPersonEmail})</h3>
                 </c:when>
                 <c:when test="${data.moderation}">
-                    <h3 class="text-center">Moderating Responses for ${isInstructor ? "Instructor" : "Student"} ${moderatedPersonName} (${moderatedPersonEmail})</h3>
+                    <div class="container">
+                        <div class="col-md-12">
+                            <h3 class="text-center">
+                              You are moderating responses for (or submitting on behalf of) ${isInstructor ? "Instructor" : "Student"} ${moderatedPersonName} (${moderatedPersonEmail})
```

Comment:
Yup the documentation does specify that indentation should be 2 spaces, and indeed we should be following that 😄 

It would be good to update the indentation for the rest of the code to using 2 spaces to avoid having the html skewed towards the right in cases where there are many nested tags. However that's a fairly monumental task. 

As most other parts of the code, for example `index.jsp` use 4 spaces for indentation, to maintain consistency, we stick to that as the amount of spaces to use for now.